src/main/java/org/ibp/api/java/impl/middleware/design/transformer/StandardVariableTransformer.java

```diff
@@ -0,0 +1,24 @@
+package org.ibp.api.java.impl.middleware.design.transformer;
+
+import org.generationcp.middleware.domain.dms.StandardVariable;
+import org.generationcp.middleware.domain.etl.MeasurementVariable;
+import org.generationcp.middleware.domain.ontology.VariableType;
+import org.springframework.stereotype.Component;
+
+@Component
+public class StandardVariableTransformer {
+
+	public MeasurementVariable convert(final StandardVariable standardVariable, final VariableType variableType) {
```

Comment:
This code seems familiar to me, is it not repeated in other places in the application?

Reply:
Removed **StandardVariableTransformer** class and overloaded **MeasurementVariableTransformer** in MIddleware for use in this module